### PR TITLE
add back button to the price database screen

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [CX-3622] 

### Description

Simulator Screen Shot - iPhone 14 Pro Max - 2023-04-14 at 09 43 02

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- add back button to the price database screen -daria

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-3622]: https://artsyproduct.atlassian.net/browse/CX-3622?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/PriceDatabase/components/PriceDatabaseSearch.tsx [hunk 1]
```diff
@@ -53,10 +53,9 @@ export const PriceDatabaseSearch: React.FC<StackScreenProps<ArtworkFilterNavigat
     <ArtsyKeyboardAvoidingView>
       <ScrollView>
         <Flex my={2}>
-          <Flex mx={2} mb={2}>
-            <Text variant="xl">Artsy Price</Text>
-            <Text variant="xl" mb={0.5}>
-              Database
+          <Flex mx={2} my={2}>
+            <Text variant="lg" mb={0.5}>
+              Artsy Price Database
```
Comment (olerichter00) [inline src/app/Scenes/PriceDatabase/components/PriceDatabaseSearch.tsx:58]: Is this change intentional?

**Review by olerichter00 — APPROVED** [review]
Thanks for adding the button 🙏

File: src/app/Scenes/PriceDatabase/components/PriceDatabaseSearch.tsx (hunk 1 above)
Comment (dariakoko) [reply src/app/Scenes/PriceDatabase/components/PriceDatabaseSearch.tsx:58]: Yesss, with the button on the screen the title should be smaller
